App.tsx

```diff
@@ -235,7 +239,14 @@ const BaseApp: React.FunctionComponent<{
 }> = ({staging, setStaging}) => {
   const colorScheme = useColorScheme();
   const {logger} = React.useContext<LoggerProps>(LoggerContext);
-  const [avalancheCenterId, setAvalancheCenterId] = React.useState((Constants.expoConfig?.extra?.avalanche_center as AvalancheCenterID) ?? 'NWAC');
+  const {preferences, setPreferences} = usePreferences();
+  const avalancheCenterId = preferences.center;
+  const setAvalancheCenterId = useCallback(
```

Comment:
Out of curiosity, why did you need to `useCallback` here and elsewhere?

Reply:
`useCallback` guarantees that the value of the function object is stable if the dependencies don't change. that helps you avoid unnecessary re-renders when you're passing the function as a prop to subcomponents. without `useCallback` the function gets a new value on every render pass.

Reply:
Gotcha, makes sense - thanks!